vhdbuilder/packer/install-dependencies.sh

```diff
@@ -165,6 +165,10 @@ if [[ $OS == $MARINER_OS_NAME ]]; then
     activateNfConntrack
 fi
 
+# installs oras for installing packages for network isolated clusters 
```

Comment:
TODO code changes here, to get an idea of what will need to be done for NIC

Reply:
cse_main.sh and install-dependencies.sh both call `installDeps`, thus this shouldn't be necessary

you could also keep it separate from `installDeps` and call `installOras` here like you're doing, as well as within `cse_main.sh`

either way you should also make sure `installDeps` is idempotent, in that it doesn't actually download anything over the wire when oras is already installed